sources/models/shared-defaults/defaults.toml

```
@@ -155,3 +155,15 @@ affected-services = ["bootstrap-containers"]
 [services.pki]
 configuration-files = []
 restart-commands = ["/usr/bin/certdog"]
+
+# DNS
+[metadata.settings.dns]
+affected-services = ["dns"]
+
+[services.dns]
+configuration-files = ["netdog"]
+restart-commands = ["netdog write-resolv-conf"]
+
+[configuration-files.netdog]
```

nit: to follow prior convention
```suggestion
[configuration-files.netdog-toml]
```

Need to make corresponding change in migration as well